r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -201,6 +212,14 @@ test_that("strftime", {
     times
   )
 
+  # with namespacing
```

Comment:
What if, instead of duplicating the whole test, you just added a column to the mutate(), like:

```
  compare_dplyr_binding(
    .input %>%
      mutate(
        x = strftime(datetime, format = formats),
        x2 = base::strftime(datetime, format = formats),
      ) %>%
      collect(),
    times
  )
```

Reply:
I think that's what I was doing initially. I could shift back to that approach. I would definitely reduce the size of some of the test files, some of which are starting to get a bit large.

Reply:
I went towards testing the namespaced version of every binding (not just for the special cases) as I thought it would be more thorough if we made sure the approach works in general, but also there is at least one test for each of the bindings (just in case some of them might require some particular treatment). Having exhaustive unit tests allowed me to identify the special cases (such as quantile, for example).

Reply:
I have now reduced the size of the unit test files (but keeping the same aim of testing at least once each namespaces version) and will mark this as resolved. Feel free to unresolve it if necessary.